src/components/EmptyStateSearch.tsx

```diff
@@ -0,0 +1,27 @@
+import { EmptyState, EmptyStateBody, EmptyStateIcon, EmptyStateVariant, Title } from '@patternfly/react-core/dist/js/components';
```

Comment:
We should import from the base of the library. autocomplete messes it up sometimes.

```suggestion
import { EmptyState, EmptyStateBody, EmptyStateIcon, EmptyStateVariant, Title } from '@patternfly/react-core';
```

Reply:
Ok, that has been changed, resolving thread.